packages/teleport/src/Desktops/Desktops.tsx

```diff
@@ -64,6 +71,26 @@ export function Desktops(props: State) {
           />
         </>
       )}
+      {isEmpty && (
+        <Empty
+          clusterId={clusterId}
+          canCreate={canCreate && !isLeafCluster}
+          onClick={() => window.open(DOC_URL)}
+          emptyStateInfo={emptyStateInfo}
+        />
+      )}
     </FeatureBox>
   );
 }
+
+const emptyStateInfo: EmptyStateInfo = {
+  title: 'Add your first Windows desktop to Teleport',
+  byline:
+    'Teleport Desktop Access provides graphical desktop access to remote Windows hosts.',
+  docsURL: DOC_URL,
+  entityType: 'desktop',
+  readOnly: {
+    title: 'No Desktops Found',
+    resource: 'applications',
```

Comment:
Should this be `applications`?

Reply:
Good catch, it should actually be `desktops` :)